### PR TITLE
using new export api for tcld

### PR DESCRIPTION
**Pull request by alice-yin**
## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Review by jumbogo — APPROVED** [review]

File: app/namespace.go [hunk 1]
```diff
@@ -1469,13 +1396,14 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				sinkNameFlag,
 			},
 			Action: func(ctx *cli.Context) error {
-				sink, err := c.getExportSink(ctx, ctx.String(NamespaceFlagName), ctx.String(sinkNameFlag.Name))
-
+				getExportSinkRes, err := c.cloudAPIClient.GetNamespaceExportSink(c.ctx, &cloudservice.GetNamespaceExportSinkRequest{
+					Namespace: ctx.String("namespace"),
```
Comment (mattkim) [inline app/namespace.go:1400]: nit: use const variable instead of string

File: app/namespace.go [hunk 2]
```diff
@@ -1489,26 +1417,32 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				RequestIDFlag,
 			},
 			Action: func(ctx *cli.Context) error {
-				namespaceName := ctx.String(NamespaceFlagName)
+				namespaceName := ctx.String(NamespaceFlag.Name)
 				sinkName := ctx.String(sinkNameFlag.Name)
-				resourceVersion, err := c.getExportSinkResourceVersion(ctx, namespaceName, sinkName)
```
Comment (mattkim) [inline app/namespace.go:1494]: are we still going to support this for older clients?

File: app/namespace.go [hunk 3]
```diff
@@ -1489,26 +1417,32 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
...
+					resourceVersion = getExportSinkRes.GetSink().GetResourceVersion()
 				}
 
-				deleteRequest := &namespaceservice.DeleteExportSinkRequest{
+				deleteRequest := &cloudservice.DeleteNamespaceExportSinkRequest{
 					Namespace:       namespaceName,
-					SinkName:        sinkName,
+					Name:            sinkName,
 					ResourceVersion: resourceVersion,
-					RequestId:       ctx.String(RequestIDFlagName),
```
Comment (mattkim) [inline app/namespace.go:1503]: do we not support sending the request id anymore?

File: app/namespace.go (hunk 2 above)
Comment (alice-yin) [reply app/namespace.go:1494]: You mean getExportSinkResourceVersion? Yes, we do. We do support all operations related to old client. But I really want to deprecate those

File: app/namespace.go [hunk 4]
```diff
@@ -1521,17 +1455,15 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
...
 			},
 			Action: func(ctx *cli.Context) error {
-				request := &namespaceservice.ListExportSinksRequest{
-					Namespace: ctx.String(NamespaceFlagName),
-					PageSize:  int32(pageSizeFlag.Value),
-					PageToken: ctx.String(pageTokenFlag.Name),
+				request := &cloudservice.GetNamespaceExportSinksRequest{
+					Namespace: ctx.String(NamespaceFlag.Name),
+					PageSize:  int32(ctx.Int(pageSizeFlag.Name)),
+					PageToken: ctx.String("page-token"),
```
Comment (mattkim) [inline app/namespace.go:1461]: same comment about using const variables.

File: app/namespace.go (hunk 3 above)
Comment (alice-yin) [reply app/namespace.go:1503]: Yes. request id is not using any more. We're using AsyncOperationId instead

**Pull request merged**